packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-model-introspection-visitor.test.ts

```diff
@@ -143,6 +147,10 @@ describe('Model Introspection Visitor', () => {
       expect((visitor as any).getType('SimpleInterface')).toEqual({ interface: 'SimpleInterface' });
     });
 
+    it('should not pascal case enum', () => {
+      expect((visitor as any).getType('camel')).toEqual({ enum: 'camel' })
```

Comment:
Can we test with a two word name like `camelCase` to ensure that `Case` is still capitalized?